packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx

```diff
@@ -38,8 +37,19 @@ const ResponseWrapper = styled('div')<{
   transition: `box-shadow 100ms ${BezierCurve.DECELERATE}, opacity 300ms ${BezierCurve.DECELERATE}`,
   display: 'flex',
   flexDirection: 'column',
-  width: Dimensions.RESPONSE_WIDTH,
-  flexShrink: 0
+  width: '100%',
+  [twoColumnResponseMediaQuery]: {
+    width: `calc(100% / 2 - ${ResponseCardDimensions.GAP}px)`
+  },
+  [threeColumnResponseMediaQuery]: {
+    width: `calc(100% / 3 - ${ResponseCardDimensions.GAP}px)`
+  },
+  [fourColumnResponseMediaQuery]: {
+    width: `calc(100% / 4 - ${ResponseCardDimensions.GAP}px)`
+  },
+  [fiveColumnResponseMediaQuery]: {
+    width: `calc(100% / 5 - ${ResponseCardDimensions.GAP}px)`
```

Comment:
as the `grid` we're using is really just a flexbox that wraps into multiple rows there's no better way of splitting it into responsive columns